### DISCUSSION TITLE
Channels to use for FCC certification of Thread based product.

### DISCUSSION THREAD

**Discussion by nikhil30081995** [openthread/openthread#8209] [Q&A] [answered]
Hello, 

We are currently trying to certify our product based for FCC standards via TUV. After contacting TUV for certification, they have told me that the device need to be tested in lower, middle and higher channel range to be called certified in all ranges. 

So as 2.4 GHz  uses the frequency band from 2401 to 2484 MHz and globally acceptable channels are 1 to 11 (2401-2473 MHz).  So i would like to know how the Thread channel 11-26 cover this range, to be more precise i would like to have info of each thread channel and the Range covered by it in 2.4GHz spectrum. 

I am looking for info. like the table given in the below link :+1: 
https://www.everythingrf.com/community/2-4-ghz-wi-fi-802-11b-g-n-channels-and-frequency-band

**Comment by zhanglongxia** [comment, marked as answer]
You can use the following formula to calculate the frequency of each Thread channel:

`Fc = 2405 + 5 (k – 11)` in megahertz, for `k = 11, 12, ..., 26`

**Comment by jwhui** [comment]
Thread uses the IEEE 802.15.4 O-QPSK 2.4 GHz PHY. See the following IEEE 802.15.4 specifications for the full details:

- IEEE 802.15.4-2006 Section 6
- IEEE 802.15.4-2015 Sections 10 and 12

**Reply by nikhil30081995** [reply]
Okay thanks for information.

**Reply by nikhil30081995** [reply]
> Thread uses the IEEE 802.15.4 O-QPSK 2.4 GHz PHY. See the following IEEE 802.15.4 specifications for the full details:
> 
>     * IEEE 802.15.4-2006 Section 6
> 
>     * IEEE 802.15.4-2015 Sections 10 and 12

And also the  "6.1.5 Transmit power" in IEEE 802.15.4-2006 is the TX power which we set on Thread device i.e in range of -20dBm to 8dBm ?

**Reply by jwhui** [reply]
Transmit power is dependent on your hardware platform, including IEEE 802.15.4 radio, whether there is an external PA, antenna, etc.

**Reply by nikhil30081995** [reply]
> Transmit power is dependent on your hardware platform, including IEEE 802.15.4 radio, whether there is an external PA, antenna, etc.
Our end device is developed using nRF52840 SOC and using PCB antenna.

**Reply by jwhui** [reply]
The transmit power also depends on what is configured via the following platform APIs:

- [otPlatRadioSetChannelMaxTransmitPower](https://openthread.io/reference/group/radio-operation#otplatradiosetchannelmaxtransmitpower)
- [otPlatRadioSetTransmitPower](https://openthread.io/reference/group/radio-config#otplatradiosettransmitpower)

Or via platform-specific build configurations and APIs.

**Reply by nikhil30081995** [reply]
> The transmit power also depends on what is configured via the following platform APIs:
> 
>     * [otPlatRadioSetChannelMaxTransmitPower](https://openthread.io/reference/group/radio-operation#otplatradiosetchannelmaxtransmitpower)
> 
>     * [otPlatRadioSetTransmitPower](https://openthread.io/reference/group/radio-config#otplatradiosettransmitpower)
> 
> 
> Or via platform-specific build configurations and APIs.

okay thanks for info.
